<launch>
    <arg name="rvizconfig" default="$(find toio_pkg)/config/setting.rviz" />
    <group ns="toio1">
        <node name="toio_bridge" pkg="toio_pkg" type="toio_bridge.py" output="screen" respawn="true">
            <param name="address" value="F0:5D:68:8E:A8:7C"/>
            <param name="iface" value="1"/>
        </node>
        <node name="tf_broadcaster" pkg="toio_pkg" type="tf_broadcaster.py" output="screen">
            <param name="frame_id" value="toio1"/>
        </node>
    </group>


    <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" output="screen"/>
</launch>